packages/rum-react/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@elastic/apm-rum-react",
+  "version": "0.0.0",
+  "description": "> TODO: description",
+  "author": "Hamid <h.jahtalab@gmail.com>",
+  "homepage": "https://www.elastic.co/guide/en/apm/agent/js-base/current/index.html",
+  "license": "MIT",
+  "main": "src/index.js",
+  "files": [
+    "src"
+  ],
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/elastic/apm-agent-rum-js.git"
+  },
+  "scripts": {
+    "script": "node ../../dev-utils/run-script.js",
+    "build": "webpack",
+    "build:dev": "webpack -w",
+    "build:e2e": "npm run script buildE2eBundles packages/rum-react/test/e2e",
+    "test:unit": "npm run script runUnitTests false packages/rum-react",
+    "test:e2e:supported": "npm run script runE2eTests packages/rum-react/wdio.conf.js",
+    "test:sauce": "npm run build:e2e && npm run script runSauceTests true packages/rum-react test:unit test:e2e:supported",
+    "karma": "karma start",
+    "test": "npm run test:sauce"
+  },
+  "bugs": {
+    "url": "https://github.com/elastic/apm-agent-rum-js/issues"
+  },
+  "dependencies": {
+    "hoist-non-react-statics": "^3.3.0"
```

Comment:
`@elastic/apm-rum` and `react-router-dom` must be a dep and `react` must be in peer dep.

Reply:
Yeah, good catch about `@elastic/apm-rum` but I think `react-router-dom` should be a peer dep as well, since we expect to use the version that the user already has installed.

Reply:
What if the user users some other router? In that case, we cannot say we cover all react apps.

Reply:
Yeah, I don't think we should say that we cover all routers, but it's more likely that users will come back to use with a feature request for those routers